pubspec.yaml

```diff
@@ -1,13 +1,14 @@
 name: observable
-version: 0.20.4+3
+version: 0.21.0
 author: Dart Team <misc@dartlang.org>
 description: Support for marking objects as observable
 homepage: https://github.com/dart-lang/observable
 environment:
-  sdk: '>=1.22.0 <2.0.0'
+  sdk: '>=2.0.0-dev.22.0 <2.0.0'
```

Comment:
Does this require -dev.22.0 for some reason? That should be in the CHANGELOG

Reply:
No reason in particular but it is the current version externally and I tested it. Version 0.20.4+3 doesn't build on dev.22.0 so there this becomes the version you need to use moving forward. WDYT?

Reply:
Changing the minimum SDK version indicates that the package became incompatible with the previous minimum SDK version. Like if the package uses generic methods, it's minimum SDK version probably got bumped to 1.21, when that syntax was introduced.

If observable 0.20.x doesn't work with some SDK version, that is not a reason to change the SDK version here.

Reply:
Sounds good. I can try to pinpoint the last version that this works on, use it as the min and pin it in the travis test matrix so we know when it breaks in the future.